vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -68,25 +68,32 @@ $global:imagesToPull += @(
     # CSI. Owner: andyzhangx (Andy Zhang)
     "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.10.0",
     "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.11.0",
+    "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.12.0",
     "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.8.0",
```

Comment:
@andyzhangx Could we remove livenessprobe:v2.10.0, csi-node-driver-registrar:v2.8.0, azuredisk-csi:v1.26.7, azuredisk-csi:v1.28.4.1-windows-hp, azuredisk-csi:v1.29.1.1-windows-hp. azurefile-csi:v1.26.9, azurefile-csi:v1.28.6-windows-hp and azurefile-csi:v1.29.1-windows-hp?

Reply:
@AbelHu already discussed with @ShiqianTao those images are still used by v20231128 release, so I added it back in next iteration in this PR.